orchestrator/types.py

```diff
@@ -202,19 +210,14 @@ def is_optional_type(t: Any, test_type: Optional[type] = None) -> bool:
     False
     >>> is_optional_type(Optional[State], State)
     True
-    >>> is_optional_type(int)
-    False
     """
-    if get_origin(t):
-        if get_origin(t) == Union and None.__class__ in get_args(t):  # type:ignore
-            for arg in get_args(t):
-                if arg is None.__class__:
-                    continue
+    union_types = [Union, UnionType] if UnionType else [Union]
```

Comment:
I would declare this once in the try:import block so that there is no None-check needed, and to not re-declare it every time. What do you think?